origin-js/src/contractInterface/marketplace/v00_adapter.js

```diff
@@ -276,19 +276,19 @@ class V00_MarkeplaceAdapter {
       } else if (event.event === 'ListingWithdrawn') {
         status = 'inactive'
       } else if (event.event === 'OfferCreated') {
-        offers[event.returnValues.offerID] = { status: 'created', event }
+        offers.push({ status: 'created', event })
```

Comment:
Couple comments:
- There can be multiple events for the same offerId (e.g. for each state of created, accepted, finalized, etc...). So adding all events to the list may not be what we want to do ? The dictionary was taking care of only keeping the latest event for each offerId.
 - How will the DApp get the list of offerIds associated with the listing ?
 - Also have you checked to make sure the DApp is not relying on doing look ups by offerId in the dict or relying on iterating over a dict rather than a list of offers ?

Overall we may want to make this change as a separate PR. Or punt on it since the graphQL rewrite will clean up a lot of the origin-js interface. Happy to discuss - feel free to ping me so that we can come up to a quick resolution on this and launch multi-units ! :)